packages/client/src/getBalances.ts

```diff
@@ -92,15 +92,21 @@ export const getBalances = async function (
     }
   });
   supportedChains.forEach((chain: any) => {
-    const contract = getAddress("zetaToken", chain.chain_name as any);
-    if (contract) {
-      tokens.push({
-        chain_id: chain.chain_id,
-        coin_type: "ERC20",
-        contract,
-        decimals: 18,
-        symbol: "WZETA",
-      });
+    const chainLabel = Object.keys(this.getChains()).find(
+      (key) => this.getChains()[key].chain_id === parseInt(chain.chain_id)
+    );
+
+    if (chainLabel) {
+      const contract = getAddress("zetaToken", chainLabel as any);
```

Comment:
typing supported chain labels may help out catching some issues in the future (eg: getEndpoint not supporting the label)

Reply:
The issue is that the label we're using on chain doesn't match the label we use in the registry of networks (zeta-chain/networks). Once we start using chain IDs for everything, I agree, typing will become feasible.